cypress.config.js

```diff
@@ -1,19 +1,19 @@
-import { defineConfig } from 'cypress';
-import { faker } from '@faker-js/faker';
-import { clear } from './dataBase';
-
+const { defineConfig } = require('cypress');
+const  faker  = require('@faker-js/faker');
+const { clear } = require('./dataBase');
 module.exports = defineConfig({
   e2e: {
     baseUrl: 'http://localhost:3000',
     setupNodeEvents(on, config) {
       on('task', {
         generateUser() {
           let randomNumber = Math.ceil(Math.random(1000) * 1000);
-          let userName = faker.name.firstName() + `${randomNumber}`;
+          let userName = faker.person.firstName() + `${randomNumber}`;
```

Comment:
The method `faker.person.firstName()` might not exist. You should verify the correct method name in the Faker.js documentation. It might be `faker.name.firstName()` instead.